core/src/main/java/org/radargun/stages/monitor/MonitorStartStage.java

```diff
@@ -75,6 +76,7 @@ private void addMonitors(AbstractMonitors monitors, Timeline timeline) {
       monitors.addMonitor(new MemoryUsageMonitor(jmxConnectionProvider, timeline));
       monitors.addMonitor(new GcMonitor(jmxConnectionProvider, timeline));
       monitors.addMonitor(new OpenFilesMonitor(jmxConnectionProvider, timeline));
+      monitors.addMonitor(new RssMonitor(timeline));
```

Comment:
You should probably add a platform check here before creating the monitor. RadarGun doesn't run on Windows yet, but that's in the plans.